webapp/locales/en.json

```diff
@@ -215,6 +215,7 @@
       },
       "table": {
         "columns": {
+          "number": "No.",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-13T10:28:14Z" title="Tuesday, August 13th 2019, 12:28:14 pm +02:00">Aug 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -215,6 +215,7 @@
       },
       "table": {
         "columns": {
+          "number": "#",
```


It could also be `No.`, if you want to use an English abbreviation.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-14T06:20:35Z" title="Wednesday, August 14th 2019, 8:20:35 am +02:00">Aug 14, 2019</time>_

---


Thanks @mattwr18 ! 🤗
Done in both places.